packages/cli/src/commands/build.ts

```diff
@@ -877,3 +931,48 @@ async function resolveNftToOutput({
     files: newFilesList,
   });
 }
+
+/**
+ * Files will only exist when `next export` was used.
+ */
+async function getNextExportStatus(dotNextDir: string) {
```

Comment:
```suggestion
async function getNextExportStatus(dotNextDir?: string) {
```

Do the return null check here